tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -3688,6 +3689,24 @@ MklLayoutRewritePass::CheckForQuantizedNodeRewrite(const Node* n) const {
   if (type_attrs_present) {
     for (auto ri = rinfo_.cbegin(); ri != rinfo_.cend(); ++ri) {
       if (n->type_string().compare(ri->name) == 0 && ri->rewrite_rule(n)) {
+	// Currently OneDNN optimization does not support int8 with native
+	// format.
+        if (NativeFormatEnabled()) {
+          static absl::once_flag once;
+	  absl::call_once(once, [] {
+#if defined(ENABLE_MKL)
+            VLOG(0) << "MklLayoutRewritePass::RewriteInfo: Do not support INT8"
+                    << "data type for native format. Please set environment "
+                    << "variable TF_ENABLE_MKL_NATIVE_FORMAT to false.";
+#else
+            VLOG(0) << "MklLayoutRewritePass::RewriteInfo: Do not support INT8 "
+	            << "data type for native format. Please switch to Intel "
+	            << "Optimized Tensorflow and set environment variable "
```

Comment:
Nit: Could you please fix the indentation? Thank you!
```suggestion
            VLOG(0) << "MklLayoutRewritePass::RewriteInfo: Do not support INT8 "
	              << "data type for native format. Please switch to Intel "
	              << "Optimized Tensorflow and set environment variable "
```

Reply:
fixed